### ISSUE TITLE
Dependency `once_cell v1.4.1` not compatible with `tokio v1.2.0` (which uses 1.5.2)

### ISSUE THREAD

**Issue by d4h0** [eyre-rs/eyre#50]
Hi,

I just started a project with `tokio`, and it turned out, the current versions of `eyre` and `tokio` are not compatible because different versions of `once_cell` are used.

Would it be possible for `eyre` to upgrade to the current version of `once_cell`?

**Comment by d4h0** [comment]
Oh, never mind. I figured out how to get my code compile without changing `eyre`... 😆

In case someone has the same problem, and stumbled upon this via Google: I disabled the `tokio` features that depend on `once_cell` (`process` and `signal`).

It seems, many other crates still depend on `once_cell v1.4.1` (even crates that depend on `tokio`), so I will close this now. But fell free to re-open, if you think it makes sense.
